pennylane/tape/reversible.py

```diff
@@ -184,7 +184,6 @@ def reversible_diff(self, idx, params, **options):
         # create a new circuit which rewinds the pre-measurement state to just after `op`,
         # applies the generator of `op`, and then plays forward back to
         # pre-measurement step
-        wires = op.wires

```

Comment:
I don't know where this came in, but it is indeed unused, as pylint complained about